sdk/loadtesting/playwright/samples/v1/javascript/set-default-authentication-mechanism/README.md

```diff
@@ -56,15 +56,15 @@ The service configuration serves to:
 
 ### Obtain region endpoint
 
-1. In the [Playwright portal](https://aka.ms/mpt/portal), copy the command under **Add region endpoint in your set up**.
+1. In the [Playwright portal](https://aka.ms/pww/docs/portal), copy the command under **Add region endpoint in your set up**.
 
     ![Set workspace endpoint](https://github.com/microsoft/playwright-testing-service/assets/12104064/d81ca629-2b23-4d34-8b70-67b6f7061a83)
 
     The endpoint URL corresponds to the workspace region. You might see a different endpoint URL in the Playwright portal, depending on the region you selected when creating the workspace.
```

Comment:
we should any reference for Playwright portal with azure portal with required screenshots.